### PR TITLE
Add basic Makefile

### PR DESCRIPTION
**Pull request by boyvanamstel**
Run `$ make` to test the project from the command line.

### CODE REVIEW

**Comment by joostvandijk** [comment]
Could you add something to the README to explain how to use this?

**Comment by boyvanamstel** [comment]
Sure!

**Comment by boyvanamstel** [comment]
@joostvandijk Done.

File: README.md [hunk 1]
```diff
@@ -101,7 +101,34 @@ Hanson is available through either [CocoaPods](http://cocoapods.org) or [Carthag
...
 1. Add `github 'blendle/Hanson'` to your `Cartfile`.
 2. Run `carthage update`.
-3. Link the framework with your target as described in [Carthage Readme](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application)
+3. Link the framework with your target as described in [Carthage Readme](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application).
+
+## Building
+
+The project obviously builds fine through Xcode, just load up `Hanson.xcodeproj` and run it.
+
+For convenience, we've included a few scripts and a `Makefile` that allow you to build Hanson from the command line and through continuous integration. They are inspired by GitHub's [Scripts to Rule Them All](https://github.com/github/scripts-to-rule-them-all) builerplate:
```
Comment (koenrh) [inline README.md]: "builerplate" --> "boilerplate"

File: README.md [hunk 2]
```diff
@@ -101,7 +101,34 @@ Hanson is available through either [CocoaPods](http://cocoapods.org) or [Carthag
 
 1. Add `github 'blendle/Hanson'` to your `Cartfile`.
 2. Run `carthage update`.
-3. Link the framework with your target as described in [Carthage Readme](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application)
+3. Link the framework with your target as described in [Carthage Readme](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application).
```
Comment (joostvandijk) [inline README.md]: Thanks for adding the period. 👍

File: README.md (hunk 1 above)
Comment (boyvanamstel) [inline README.md]: Fixed.

**Pull request merged**